### PR TITLE
Adding SSp_Detector recipe

### PR DESCRIPTION
**Pull request by sjg2203**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sjg2203** [comment]
@conda-forge-admin (@ocefpaf) & @conda-forge/help-python @conda-forge/staged-recipes Ready for review!

File: recipes/SSp_Detector/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  home: https://github.com/sjg2203/SSp_Detector
+  summary: Sleep spindles detector
+  license: Apache-2.0
+  license_file: LICENSE
+  doc_url: https://github.com/sjg2203/SSp_Detector/blob/main/README.md
+  dev_url: https://github.com/sjg2203/SSp_Detector
+
+extra:
+  recipe-maintainers:
+    - sjg2203
```
Comment (ocefpaf) [inline recipes/SSp_Detector/meta.yaml:43]: You are still removing the pypromice recipe. Let me rebase this for you.

File: recipes/SSp_Detector/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{% set name = "SSp_Detector" %}
+{% set version = "2023.10.25rc1" %}
```
Comment (ocefpaf) [inline recipes/SSp_Detector/meta.yaml:2]: This still falls under CFEP-05. See https://github.com/conda-forge/cfep/blob/main/cfep-05.md

You can ask upstream for a stable release or publish under a dev label. See the instructions for the latter in the doc above.

File: recipes/SSp_Detector/meta.yaml (hunk 1 above)
Comment (sjg2203) [reply recipes/SSp_Detector/meta.yaml:43]: Apologies, the example I saw was removing pypromice. Thanks for the help

File: recipes/SSp_Detector/meta.yaml (hunk 2 above)
Comment (sjg2203) [reply recipes/SSp_Detector/meta.yaml]: Version number updated for stable release

**Pull request merged**